### ISSUE TITLE
-monochrome and -compress Group4 output odd tiff

### ISSUE THREAD

**Issue by t246246** [ImageMagick/ImageMagick#1627]
### Prerequisites

- [x ] I have written a descriptive issue title
- [x ] I have verified that I am using the latest version of ImageMagick
- [x ] I have searched [open](https://github.com/ImageMagick/ImageMagick/issues) and [closed](https://github.com/ImageMagick/ImageMagick/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description

I'm not sure this is a bug, but there is a request to open issue.  See https://github.com/dlemstra/Magick.NET/issues/480

### Steps to Reproduce

Save attached png file as x.png.
```
$ magick x.png -compress Group4 x.tif
```
output is black text on white background.
whereas
```
$ magick x.png -monochrome -compress Group4 x.tif
```
output turns out to all black.

### System Configuration

- ImageMagick version: 7.0.8-50 Q16 HDRI
![x](https://user-images.githubusercontent.com/19387062/60711525-1d57bc80-9f50-11e9-8dc2-d43bfd1465e1.png)

- Environment (Operating system, version and so on): Windows 7 (msys2)
- Additional information: Also tried with 7.0.8-51 from standard command line on the same Windows.

**Comment by t246246** [comment]
So far as I tried, this happened input file is black and white only but also has alpha.

$ magick x.png -format '%c' histogram:info:-
      1990: (  0,  0,  0,255) #000000FF black
       642: (  0,  0,  0,136) #00000088 srgba(0,0,0,0.533333)
       579: (  0,  0,  0, 68) #00000044 srgba(0,0,0,0.266667)
       431: (  0,  0,  0,187) #000000BB srgba(0,0,0,0.733333)
       189: (  0,  0,  0, 17) #00000011 srgba(0,0,0,0.0666667)
       160: (  0,  0,  0,238) #000000EE srgba(0,0,0,0.933333)
       148: (  0,  0,  0,204) #000000CC srgba(0,0,0,0.8)
       148: (  0,  0,  0, 51) #00000033 srgba(0,0,0,0.2)
       145: (  0,  0,  0,102) #00000066 srgba(0,0,0,0.4)
       128: (  0,  0,  0, 34) #00000022 srgba(0,0,0,0.133333)
       125: (  0,  0,  0,153) #00000099 srgba(0,0,0,0.6)
       123: (  0,  0,  0,221) #000000DD srgba(0,0,0,0.866667)
       105: (  0,  0,  0,170) #000000AA srgba(0,0,0,0.666667)
        58: (  0,  0,  0, 85) #00000055 srgba(0,0,0,0.333333)
        57: (  0,  0,  0,119) #00000077 srgba(0,0,0,0.466667)
   5838722: (255,255,255,  0) #FFFFFF00 srgba(255,255,255,0)
[/code]
However if input is
[code]
$ magick logo: \( +clone -negate -colorspace HSV -separate -delete 0,1 \) \( -clone 0 -dither riemersma -quantize gray -colors 2 \) -delete 0 +swap -compose copy_opacity -composite -colorspace Gray y.png
[/code]
[code]
$ magick y.png -format '%c' histogram:info:-                                         12280: (  0,  0,  0,221) #000000DD srgba(0,0,0,0.866667)
      7150: (  0,  0,  0,251) #000000FB srgba(0,0,0,0.984314)
      2928: (  0,  0,  0,255) #000000FF black
      2773: (  0,  0,  0,223) #000000DF srgba(0,0,0,0.87451)
      1357: (  0,  0,  0,217) #000000D9 srgba(0,0,0,0.85098)
       822: (  0,  0,  0,219) #000000DB srgba(0,0,0,0.858824)
       748: (  0,  0,  0,186) #000000BA srgba(0,0,0,0.729412)
       650: (  0,  0,  0,151) #00000097 srgba(0,0,0,0.592157)
       645: (  0,  0,  0,216) #000000D8 srgba(0,0,0,0.847059)
       417: (  0,  0,  0,235) #000000EB srgba(0,0,0,0.921569)
       402: (  0,  0,  0,218) #000000DA srgba(0,0,0,0.854902)
       314: (  0,  0,  0,204) #000000CC srgba(0,0,0,0.8)
       305: (  0,  0,  0,117) #00000075 srgba(0,0,0,0.458824)
       299: (  0,  0,  0,224) #000000E0 srgba(0,0,0,0.878431)
       293: (  0,  0,  0,225) #000000E1 srgba(0,0,0,0.882353)
       263: (  0,  0,  0,242) #000000F2 srgba(0,0,0,0.94902)
       247: (  0,  0,  0,249) #000000F9 srgba(0,0,0,0.976471)
       246: (  0,  0,  0,220) #000000DC srgba(0,0,0,0.862745)
       241: (  0,  0,  0,157) #0000009D srgba(0,0,0,0.615686)
       194: (  0,  0,  0,208) #000000D0 srgba(0,0,0,0.815686)
       193: (  0,  0,  0,243) #000000F3 srgba(0,0,0,0.952941)
       188: (  0,  0,  0,230) #000000E6 srgba(0,0,0,0.901961)
       188: (  0,  0,  0,246) #000000F6 srgba(0,0,0,0.964706)
       165: (  0,  0,  0,172) #000000AC srgba(0,0,0,0.67451)
       156: (  0,  0,  0,125) #0000007D srgba(0,0,0,0.490196)
       154: (  0,  0,  0,112) #00000070 srgba(0,0,0,0.439216)
       146: (  0,  0,  0,182) #000000B6 srgba(0,0,0,0.713725)
       146: (  0,  0,  0, 99) #00000063 srgba(0,0,0,0.388235)
       143: (  0,  0,  0,245) #000000F5 srgba(0,0,0,0.960784)
       137: (  0,  0,  0,239) #000000EF srgba(0,0,0,0.937255)
       134: (  0,  0,  0,236) #000000EC srgba(0,0,0,0.92549)
       133: (  0,  0,  0,141) #0000008D srgba(0,0,0,0.552941)
       126: (  0,  0,  0,231) #000000E7 srgba(0,0,0,0.905882)
       120: (  0,  0,  0,229) #000000E5 srgba(0,0,0,0.898039)
       116: (  0,  0,  0,201) #000000C9 srgba(0,0,0,0.788235)
       116: (  0,  0,  0, 66) #00000042 srgba(0,0,0,0.258824)
       114: (  0,  0,  0, 97) #00000061 srgba(0,0,0,0.380392)
       112: (  0,  0,  0,191) #000000BF srgba(0,0,0,0.74902)
       111: (  0,  0,  0,227) #000000E3 srgba(0,0,0,0.890196)
       108: (  0,  0,  0,149) #00000095 srgba(0,0,0,0.584314)
       107: (  0,  0,  0,200) #000000C8 srgba(0,0,0,0.784314)
       107: (  0,  0,  0,228) #000000E4 srgba(0,0,0,0.894118)
       105: (  0,  0,  0,128) #00000080 srgba(0,0,0,0.501961)
       101: (  0,  0,  0,175) #000000AF srgba(0,0,0,0.686275)
       100: (  0,  0,  0,203) #000000CB srgba(0,0,0,0.796078)
        98: (  0,  0,  0,241) #000000F1 srgba(0,0,0,0.945098)
        97: (  0,  0,  0,244) #000000F4 srgba(0,0,0,0.956863)
        94: (  0,  0,  0,213) #000000D5 srgba(0,0,0,0.835294)
        92: (  0,  0,  0,152) #00000098 srgba(0,0,0,0.596078)
        92: (  0,  0,  0,237) #000000ED srgba(0,0,0,0.929412)
        90: (  0,  0,  0, 83) #00000053 srgba(0,0,0,0.32549)
        90: (  0,  0,  0,107) #0000006B srgba(0,0,0,0.419608)
        87: (  0,  0,  0,207) #000000CF srgba(0,0,0,0.811765)
        86: (  0,  0,  0, 64) #00000040 srgba(0,0,0,0.25098)
        86: (  0,  0,  0,181) #000000B5 srgba(0,0,0,0.709804)
        81: (  0,  0,  0,115) #00000073 srgba(0,0,0,0.45098)
        78: (  0,  0,  0,143) #0000008F srgba(0,0,0,0.560784)
        75: (  0,  0,  0,184) #000000B8 srgba(0,0,0,0.721569)
        74: (  0,  0,  0,159) #0000009F srgba(0,0,0,0.623529)
        72: (  0,  0,  0,202) #000000CA srgba(0,0,0,0.792157)
        72: (  0,  0,  0,238) #000000EE srgba(0,0,0,0.933333)
        70: (  0,  0,  0, 80) #00000050 srgba(0,0,0,0.313725)
        70: (  0,  0,  0,180) #000000B4 srgba(0,0,0,0.705882)
        69: (  0,  0,  0,155) #0000009B srgba(0,0,0,0.607843)
        68: (  0,  0,  0,214) #000000D6 srgba(0,0,0,0.839216)
        68: (  0,  0,  0,176) #000000B0 srgba(0,0,0,0.690196)
        67: (  0,  0,  0,193) #000000C1 srgba(0,0,0,0.756863)
        67: (  0,  0,  0,240) #000000F0 srgba(0,0,0,0.941176)
        67: (  0,  0,  0,169) #000000A9 srgba(0,0,0,0.662745)
        66: (  0,  0,  0,199) #000000C7 srgba(0,0,0,0.780392)
        64: (  0,  0,  0,209) #000000D1 srgba(0,0,0,0.819608)
        63: (  0,  0,  0,183) #000000B7 srgba(0,0,0,0.717647)
        62: (  0,  0,  0,198) #000000C6 srgba(0,0,0,0.776471)
        60: (  0,  0,  0,232) #000000E8 srgba(0,0,0,0.909804)
        60: (  0,  0,  0,212) #000000D4 srgba(0,0,0,0.831373)
        56: (  0,  0,  0,171) #000000AB srgba(0,0,0,0.670588)
        56: (  0,  0,  0, 88) #00000058 srgba(0,0,0,0.345098)
        55: (  0,  0,  0,165) #000000A5 srgba(0,0,0,0.647059)
        55: (  0,  0,  0, 96) #00000060 srgba(0,0,0,0.376471)
        53: (  0,  0,  0,109) #0000006D srgba(0,0,0,0.427451)
        51: (  0,  0,  0, 94) #0000005E srgba(0,0,0,0.368627)
        50: (  0,  0,  0,187) #000000BB srgba(0,0,0,0.733333)
        50: (  0,  0,  0,247) #000000F7 srgba(0,0,0,0.968627)
        49: (  0,  0,  0, 91) #0000005B srgba(0,0,0,0.356863)
        44: (  0,  0,  0,168) #000000A8 srgba(0,0,0,0.658824)
        42: (  0,  0,  0,233) #000000E9 srgba(0,0,0,0.913725)
        42: (  0,  0,  0, 74) #0000004A srgba(0,0,0,0.290196)
        41: (  0,  0,  0,166) #000000A6 srgba(0,0,0,0.65098)
        41: (  0,  0,  0,192) #000000C0 srgba(0,0,0,0.752941)
        38: (  0,  0,  0,162) #000000A2 srgba(0,0,0,0.635294)
        38: (  0,  0,  0,121) #00000079 srgba(0,0,0,0.47451)
        38: (  0,  0,  0,170) #000000AA srgba(0,0,0,0.666667)
        36: (  0,  0,  0,206) #000000CE srgba(0,0,0,0.807843)
        36: (  0,  0,  0,161) #000000A1 srgba(0,0,0,0.631373)
        35: (  0,  0,  0,134) #00000086 srgba(0,0,0,0.52549)
        35: (  0,  0,  0, 41) #00000029 srgba(0,0,0,0.160784)
        35: (  0,  0,  0, 48) #00000030 srgba(0,0,0,0.188235)
        35: (  0,  0,  0, 52) #00000034 srgba(0,0,0,0.203922)
        34: (  0,  0,  0, 85) #00000055 srgba(0,0,0,0.333333)
        33: (  0,  0,  0,197) #000000C5 srgba(0,0,0,0.772549)
        33: (  0,  0,  0,195) #000000C3 srgba(0,0,0,0.764706)
        33: (  0,  0,  0,104) #00000068 srgba(0,0,0,0.407843)
        31: (  0,  0,  0,188) #000000BC srgba(0,0,0,0.737255)
        29: (  0,  0,  0, 49) #00000031 srgba(0,0,0,0.192157)
        29: (  0,  0,  0, 62) #0000003E srgba(0,0,0,0.243137)
        27: (  0,  0,  0,105) #00000069 srgba(0,0,0,0.411765)
        25: (  0,  0,  0,174) #000000AE srgba(0,0,0,0.682353)
        25: (  0,  0,  0, 58) #0000003A srgba(0,0,0,0.227451)
        25: (  0,  0,  0,101) #00000065 srgba(0,0,0,0.396078)
        25: (  0,  0,  0,130) #00000082 srgba(0,0,0,0.509804)
        23: (  0,  0,  0, 72) #00000048 srgba(0,0,0,0.282353)
        21: (  0,  0,  0, 78) #0000004E srgba(0,0,0,0.305882)
        21: (  0,  0,  0, 24) #00000018 srgba(0,0,0,0.0941176)
        20: (  0,  0,  0, 33) #00000021 srgba(0,0,0,0.129412)
        20: (  0,  0,  0, 56) #00000038 srgba(0,0,0,0.219608)
        19: (  0,  0,  0, 32) #00000020 srgba(0,0,0,0.12549)
        18: (  0,  0,  0,178) #000000B2 srgba(0,0,0,0.698039)
        17: (  0,  0,  0, 69) #00000045 srgba(0,0,0,0.270588)
        17: (  0,  0,  0,177) #000000B1 srgba(0,0,0,0.694118)
        15: (  0,  0,  0, 40) #00000028 srgba(0,0,0,0.156863)
        13: (  0,  0,  0,215) #000000D7 srgba(0,0,0,0.843137)
         9: (  0,  0,  0, 45) #0000002D srgba(0,0,0,0.176471)
         9: (  0,  0,  0,205) #000000CD srgba(0,0,0,0.803922)
         8: (  0,  0,  0, 36) #00000024 srgba(0,0,0,0.141176)
         6: (  0,  0,  0,189) #000000BD srgba(0,0,0,0.741176)
         6: (  0,  0,  0,179) #000000B3 srgba(0,0,0,0.701961)
         5: (  0,  0,  0, 18) #00000012 srgba(0,0,0,0.0705882)
         5: (  0,  0,  0, 16) #00000010 srgba(0,0,0,0.0627451)
         5: (  0,  0,  0, 29) #0000001D srgba(0,0,0,0.113725)
         3: (  0,  0,  0, 12) #0000000C srgba(0,0,0,0.0470588)
         1: (  0,  0,  0, 77) #0000004D srgba(0,0,0,0.301961)
         1: (  0,  0,  0,118) #00000076 srgba(0,0,0,0.462745)
    256244: (255,255,255,  0) #FFFFFF00 srgba(255,255,255,0)
      1359: (255,255,255,201) #FFFFFFC9 srgba(255,255,255,0.788235)
       953: (255,255,255,151) #FFFFFF97 srgba(255,255,255,0.592157)
       725: (255,255,255,221) #FFFFFFDD srgba(255,255,255,0.866667)
       356: (255,255,255, 16) #FFFFFF10 srgba(255,255,255,0.0627451)
       356: (255,255,255,200) #FFFFFFC8 srgba(255,255,255,0.784314)
       340: (255,255,255, 64) #FFFFFF40 srgba(255,255,255,0.25098)
       313: (255,255,255, 18) #FFFFFF12 srgba(255,255,255,0.0705882)
       312: (255,255,255, 12) #FFFFFF0C srgba(255,255,255,0.0470588)
       288: (255,255,255, 66) #FFFFFF42 srgba(255,255,255,0.258824)
       285: (255,255,255, 33) #FFFFFF21 srgba(255,255,255,0.129412)
       255: (255,255,255,117) #FFFFFF75 srgba(255,255,255,0.458824)
       245: (255,255,255, 32) #FFFFFF20 srgba(255,255,255,0.12549)
       228: (255,255,255, 48) #FFFFFF30 srgba(255,255,255,0.188235)
       189: (255,255,255, 24) #FFFFFF18 srgba(255,255,255,0.0941176)
       184: (255,255,255, 49) #FFFFFF31 srgba(255,255,255,0.192157)
       164: (255,255,255, 80) #FFFFFF50 srgba(255,255,255,0.313725)
       163: (255,255,255, 99) #FFFFFF63 srgba(255,255,255,0.388235)
       156: (255,255,255,247) #FFFFFFF7 srgba(255,255,255,0.968627)
       140: (255,255,255,112) #FFFFFF70 srgba(255,255,255,0.439216)
       139: (255,255,255, 11) #FFFFFF0B srgba(255,255,255,0.0431373)
       139: (255,255,255, 83) #FFFFFF53 srgba(255,255,255,0.32549)
       139: (255,255,255, 41) #FFFFFF29 srgba(255,255,255,0.160784)
       125: (255,255,255, 97) #FFFFFF61 srgba(255,255,255,0.380392)
       119: (255,255,255,195) #FFFFFFC3 srgba(255,255,255,0.764706)
       118: (255,255,255, 96) #FFFFFF60 srgba(255,255,255,0.376471)
       111: (255,255,255, 88) #FFFFFF58 srgba(255,255,255,0.345098)
       109: (255,255,255,204) #FFFFFFCC srgba(255,255,255,0.8)
        95: (255,255,255, 52) #FFFFFF34 srgba(255,255,255,0.203922)
        92: (255,255,255,107) #FFFFFF6B srgba(255,255,255,0.419608)
        92: (255,255,255, 62) #FFFFFF3E srgba(255,255,255,0.243137)
        86: (255,255,255, 45) #FFFFFF2D srgba(255,255,255,0.176471)
        84: (255,255,255, 74) #FFFFFF4A srgba(255,255,255,0.290196)
        83: (255,255,255,155) #FFFFFF9B srgba(255,255,255,0.607843)
        83: (255,255,255, 40) #FFFFFF28 srgba(255,255,255,0.156863)
        81: (255,255,255,128) #FFFFFF80 srgba(255,255,255,0.501961)
        79: (255,255,255, 29) #FFFFFF1D srgba(255,255,255,0.113725)
        78: (255,255,255, 56) #FFFFFF38 srgba(255,255,255,0.219608)
        77: (255,255,255,199) #FFFFFFC7 srgba(255,255,255,0.780392)
        77: (255,255,255,152) #FFFFFF98 srgba(255,255,255,0.596078)
        75: (255,255,255,166) #FFFFFFA6 srgba(255,255,255,0.65098)
        72: (255,255,255,182) #FFFFFFB6 srgba(255,255,255,0.713725)
        68: (255,255,255, 58) #FFFFFF3A srgba(255,255,255,0.227451)
        68: (255,255,255,184) #FFFFFFB8 srgba(255,255,255,0.721569)
        67: (255,255,255,172) #FFFFFFAC srgba(255,255,255,0.67451)
        66: (255,255,255,115) #FFFFFF73 srgba(255,255,255,0.45098)
        65: (255,255,255, 36) #FFFFFF24 srgba(255,255,255,0.141176)
        59: (255,255,255, 78) #FFFFFF4E srgba(255,255,255,0.305882)
        57: (255,255,255,161) #FFFFFFA1 srgba(255,255,255,0.631373)
        56: (255,255,255,186) #FFFFFFBA srgba(255,255,255,0.729412)
        51: (255,255,255,189) #FFFFFFBD srgba(255,255,255,0.741176)
        49: (255,255,255,203) #FFFFFFCB srgba(255,255,255,0.796078)
        48: (255,255,255, 94) #FFFFFF5E srgba(255,255,255,0.368627)
        48: (255,255,255, 91) #FFFFFF5B srgba(255,255,255,0.356863)
        48: (255,255,255,157) #FFFFFF9D srgba(255,255,255,0.615686)
        46: (255,255,255,180) #FFFFFFB4 srgba(255,255,255,0.705882)
        45: (255,255,255, 69) #FFFFFF45 srgba(255,255,255,0.270588)
        42: (255,255,255,143) #FFFFFF8F srgba(255,255,255,0.560784)
        41: (255,255,255, 85) #FFFFFF55 srgba(255,255,255,0.333333)
        40: (255,255,255,149) #FFFFFF95 srgba(255,255,255,0.584314)
        40: (255,255,255,207) #FFFFFFCF srgba(255,255,255,0.811765)
        37: (255,255,255,169) #FFFFFFA9 srgba(255,255,255,0.662745)
        37: (255,255,255,147) #FFFFFF93 srgba(255,255,255,0.576471)
        35: (255,255,255,232) #FFFFFFE8 srgba(255,255,255,0.909804)
        34: (255,255,255,109) #FFFFFF6D srgba(255,255,255,0.427451)
        33: (255,255,255,205) #FFFFFFCD srgba(255,255,255,0.803922)
        33: (255,255,255,159) #FFFFFF9F srgba(255,255,255,0.623529)
        32: (255,255,255,118) #FFFFFF76 srgba(255,255,255,0.462745)
        32: (255,255,255,178) #FFFFFFB2 srgba(255,255,255,0.698039)
        31: (255,255,255, 46) #FFFFFF2E srgba(255,255,255,0.180392)
        30: (255,255,255, 77) #FFFFFF4D srgba(255,255,255,0.301961)
        29: (255,255,255,101) #FFFFFF65 srgba(255,255,255,0.396078)
        28: (255,255,255,183) #FFFFFFB7 srgba(255,255,255,0.717647)
        27: (255,255,255,202) #FFFFFFCA srgba(255,255,255,0.792157)
        27: (255,255,255, 72) #FFFFFF48 srgba(255,255,255,0.282353)
        26: (255,255,255,224) #FFFFFFE0 srgba(255,255,255,0.878431)
        26: (255,255,255,193) #FFFFFFC1 srgba(255,255,255,0.756863)
        25: (255,255,255,105) #FFFFFF69 srgba(255,255,255,0.411765)
        24: (255,255,255,125) #FFFFFF7D srgba(255,255,255,0.490196)
        24: (255,255,255,214) #FFFFFFD6 srgba(255,255,255,0.839216)
        23: (255,255,255,175) #FFFFFFAF srgba(255,255,255,0.686275)
        23: (255,255,255, 93) #FFFFFF5D srgba(255,255,255,0.364706)
        22: (255,255,255,198) #FFFFFFC6 srgba(255,255,255,0.776471)
        20: (255,255,255,206) #FFFFFFCE srgba(255,255,255,0.807843)
        20: (255,255,255,188) #FFFFFFBC srgba(255,255,255,0.737255)
        19: (255,255,255,104) #FFFFFF68 srgba(255,255,255,0.407843)
        19: (255,255,255,197) #FFFFFFC5 srgba(255,255,255,0.772549)
        19: (255,255,255,141) #FFFFFF8D srgba(255,255,255,0.552941)
        19: (255,255,255,223) #FFFFFFDF srgba(255,255,255,0.87451)
        18: (255,255,255,191) #FFFFFFBF srgba(255,255,255,0.74902)
        18: (255,255,255,217) #FFFFFFD9 srgba(255,255,255,0.85098)
        15: (255,255,255,219) #FFFFFFDB srgba(255,255,255,0.858824)
        14: (255,255,255, 31) #FFFFFF1F srgba(255,255,255,0.121569)
        14: (255,255,255,121) #FFFFFF79 srgba(255,255,255,0.47451)
        14: (255,255,255,171) #FFFFFFAB srgba(255,255,255,0.670588)
        14: (255,255,255,255) #FFFFFFFF white
        13: (255,255,255,181) #FFFFFFB5 srgba(255,255,255,0.709804)
        13: (255,255,255,177) #FFFFFFB1 srgba(255,255,255,0.694118)
        12: (255,255,255,134) #FFFFFF86 srgba(255,255,255,0.52549)
        11: (255,255,255,208) #FFFFFFD0 srgba(255,255,255,0.815686)
        10: (255,255,255,130) #FFFFFF82 srgba(255,255,255,0.509804)
         8: (255,255,255,176) #FFFFFFB0 srgba(255,255,255,0.690196)
         7: (255,255,255,179) #FFFFFFB3 srgba(255,255,255,0.701961)
         7: (255,255,255,220) #FFFFFFDC srgba(255,255,255,0.862745)
         7: (255,255,255,168) #FFFFFFA8 srgba(255,255,255,0.658824)
         6: (255,255,255,170) #FFFFFFAA srgba(255,255,255,0.666667)
         6: (255,255,255,187) #FFFFFFBB srgba(255,255,255,0.733333)
         5: (255,255,255,216) #FFFFFFD8 srgba(255,255,255,0.847059)
         3: (255,255,255,165) #FFFFFFA5 srgba(255,255,255,0.647059)
         2: (255,255,255,209) #FFFFFFD1 srgba(255,255,255,0.819608)
         2: (255,255,255,225) #FFFFFFE1 srgba(255,255,255,0.882353)
         2: (255,255,255,215) #FFFFFFD7 srgba(255,255,255,0.843137)
         2: (255,255,255,239) #FFFFFFEF srgba(255,255,255,0.937255)
         2: (255,255,255,212) #FFFFFFD4 srgba(255,255,255,0.831373)
         1: (255,255,255,213) #FFFFFFD5 srgba(255,255,255,0.835294)
         1: (255,255,255,162) #FFFFFFA2 srgba(255,255,255,0.635294)
[/code]
It does not happen.
[code]
$ magick y.png -compress Group4 y1.tif
$ magick y.png -monochrome -compress Group4 y2.tif
$ magick compare y1.tif y2.tif -metric PSNR -format '%c' NUL:
inf
[/code]

I guess -monochrome check the values of alpha while calculating the result pixel value.  Is it an intended behavior?

**Comment by richmarisa** [comment]
Converting a white image to G4 tiff produces an all black image (incorrect):
```
convert \( -size 2500x3300 xc:white -density 300x300 \) -compress group4 outputa.tif
magick outputa.tif -format '%c' histogram:info:- | more
   8250000: (  0,  0,  0) #000000 gray(0)
```
 
But the G4 tiff image is correct if the first pixel is set to black. 
```
convert \( -size 2500x3300 xc:white -density 300x300 \) -fill black -draw 'point 0,0' -compress group4 outputb.tif
magick outputb.tif -format '%c' histogram:info:- | more
         1: (  0,  0,  0) #000000 gray(0)
   8249999: (255,255,255) #FFFFFF gray(255)
```

ImageMagick 7.0.8-53 Q16 x86_64 2019-07-09

**Comment by t246246** [comment]
@richmarisa

Your case seems to be fixed in recent changes, I checked at 7.0.9-2 Q16 version.
